test/test-pages/build.sh

```diff
@@ -0,0 +1,11 @@
+pushd test/test-pages/
```

Comment:
Do you think we should add the node shebang to this file?

Reply:
I don't *think* so, since this is intended to be a pure shell script. i might be wrong about how that works though, I'm still a little rough around the edges with shell and bash.

Reply:
Same here; I was just wondering since it's listed in in the scripts section of `package.json`
If it's working though, the no sense in 'fixing' it so to speak